### PR TITLE
Play docs

### PR DESCRIPTION
**Pull request by qoega**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

### CODE REVIEW

File: docs/en/getting_started/playground.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# ClickHouse Playground
+
+ClickHouse Playground allows users to run ClickHouse queries instantly, without setting up their own server or cluster.
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
[ClickHouse Playground](https://play.clickhouse.tech?file=welcome) allows people to experiment with ClickHouse by running queries instantly, without setting up their own server or cluster.
```

File: docs/en/getting_started/playground.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Several example datasets are available in Playground as well as sample queries that show ClickHouse features.
+
+The queries are run as readonly user. This implies some limitations:
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
The queries are executed as read-only user. This implies some limitations:
```

File: docs/en/getting_started/playground.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- DDL queries are not allowed
+- INSERT queries are not allowed
+
+Following settings are also set:
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
The following settings are also enforced:
```

File: docs/en/getting_started/playground.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- `max_result_bytes=10485760` 
+- `max_result_rows=2000`
+- `result_overflow_mode=break`
+- `max_execution_time=60000`
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]: Links to documentation about these settings?

File: docs/en/getting_started/playground.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+ClickHouse Playground gives the experience of m2.small
+[Managed Service for ClickHouse](https://cloud.yandex.com/services/managed-clickhouse)
+instance hosted in [Yandex.Cloud](https://cloud.yandex.com/).
+More information about [cloud providers](../commercial/cloud.md).
+
+ClickHouse Playground web interface makes requests via ClickHouse [HTTP API](../interfaces/http.md).
+Backend is just a ClickHouse cluster.
+ClickHouse HTTP endpoint is is also available as a part of Playground.
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
ClickHouse HTTPS endpoint is is also available as a part of Playground.
```

File: docs/en/getting_started/playground.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- `result_overflow_mode=break`
+- `max_execution_time=60000`
+
+ClickHouse Playground gives the experience of m2.small
+[Managed Service for ClickHouse](https://cloud.yandex.com/services/managed-clickhouse)
+instance hosted in [Yandex.Cloud](https://cloud.yandex.com/).
+More information about [cloud providers](../commercial/cloud.md).
+
+ClickHouse Playground web interface makes requests via ClickHouse [HTTP API](../interfaces/http.md).
+Backend is just a ClickHouse cluster.
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
Playground backend is just a ClickHouse cluster without any additional server-side application.
```

File: docs/en/getting_started/playground.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+ClickHouse HTTP endpoint is is also available as a part of Playground.
+
+You can make queries to playground using `curl/wget` or set up a connection using [JDBC](../interfaces/jdbc.md)/[ODBC](../interfaces/odbc.md) drivers.
```
Comment (blinkov) [inline docs/en/getting_started/playground.md]:
```suggestion
You can make queries to playground using any HTTP client, for example [curl](https://curl.haxx.se) or [wget](https://curl.haxx.se), or set up a connection using [JDBC](../interfaces/jdbc.md) or [ODBC](../interfaces/odbc.md) drivers.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/ru/getting_started/playground.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+Запросы выполняются под пользователем с правами `readonly` для которого есть следующие ограничения:
+- запрещены DDL запросы
+- запроещены INSERT запросы
+
+Также установлены следующие опции:
+- `max_result_bytes=10485760` 
+- `max_result_rows=2000`
+- `result_overflow_mode=break`
+- `max_execution_time=60000`
```
Comment (qoega) [inline docs/ru/getting_started/playground.md]:
```suggestion
- [`max_result_bytes=10485760`](../operations/settings/query_complexity/#max-result-bytes) 
- [`max_result_rows=2000`](../operations/settings/query_complexity/#setting-max_result_rows) 
- [`result_overflow_mode=break`](../operations/settings/query_complexity/#result-overflow-mode)
- [`max_execution_time=60000`](../operations/settings/query_complexity/#max-execution-time)
```

File: docs/en/getting_started/playground.md (hunk 4 above)
Comment (qoega) [inline docs/en/getting_started/playground.md]:
```suggestion
- [`max_result_bytes=10485760`](../operations/settings/query_complexity/#max-result-bytes) 
- [`max_result_rows=2000`](../operations/settings/query_complexity/#setting-max_result_rows) 
- [`result_overflow_mode=break`](../operations/settings/query_complexity/#result-overflow-mode)
- [`max_execution_time=60000`](../operations/settings/query_complexity/#max-execution-time)
```

**Pull request merged**